row_009_s2.py

```
@@ -0,0 +1,120 @@
+        
+    except Exception as e:
+        logger.error(f"Error generating masks: {str(e)}")
+        return jsonify({'error': str(e)}), 500
+
+@app.route('/health', methods=['GET'])
+def health_check():
+    return jsonify({'status': 'healthy', 'model_loaded': mask_generator is not None})
+
+if __name__ == '__main__':
+    # Initialize the mask generator once when the server starts
+    sam_checkpoint = sys.argv[1]
```

Ruff (0.13.2)

Undefined name `sys`

(F821)